app/components/candidate_interface/references_review_component.rb

```diff
@@ -91,7 +91,7 @@ def name_row(reference)
     def email_row(reference)
       {
         key: 'Email address',
-        value: reference.email_address,
+        value: reference.email_address || 'Not entered',
```

Comment:
In theory this isn't needed, as there will never be a blank/nil name, is that right? Probably doesn't do any harm to leave it here though.

Reply:
Yeah I did think this but it felt kinda wrong to exclude this row aesthetically will leave be

Reply:
Hey @frankieroberto, I managed to update the continue button and the row ordering. 

The "change" is a bit more difficult as it appears to be part of the ViewComponent class. Igt woul be possible to remove it and perhaps put a hard link in the place of not entered to redirect in the same way. I think should be handled in a different PR though as it will likely mean the associated tests will have to be changed to accomodate this.

I'm not to sure as to your point about routing? There is a validation on this page to prevent a blank entry?